code/game/area/Space Station 13 areas.dm

```diff
@@ -1222,7 +1222,7 @@ proc/process_adminbus_teleport_locs()
 	icon_state = "showroom"
 
 /area/assembly/assembly_line //Derelict Assembly Line
```

Comment:
Yes. Change this back to what it was and add an `/area/crew_quarters/shop` with just the `name` and `icon_state` variables
Just steal the `icon_state` from a nearby but not adjacent area if you don't want to actually draw one